.vscode/launch.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@superzordon - can we update the `.gitignore` in this repo to exclude vscode stuff? unless you think there's a legitimate reason to include it in the repo.